components/ftx/browser/ftx_service.h

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_FTX_BROWSER_FTX_SERVICE_H_
+#define BRAVE_COMPONENTS_FTX_BROWSER_FTX_SERVICE_H_
+
+#include <list>
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "base/callback_forward.h"
+#include "base/containers/queue.h"
+#include "base/files/file_path.h"
+#include "base/macros.h"
+#include "base/memory/scoped_refptr.h"
+#include "base/memory/weak_ptr.h"
+#include "base/scoped_observer.h"
```

Comment:
this header is not used in this file and others (like base/callback.h) are missing. See https://github.com/include-what-you-use/include-what-you-use/blob/master/docs/WhyIWYU.md

Reply:
fixed via https://github.com/brave/brave-core/compare/6f4c503619e82a440ef7db9782288e922c37d5e4..f4d887c0baf39543331107f2e22f9b0ad4986c02